src/BaGet.UI/package.json

```diff
@@ -49,5 +52,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "office-ui-fabric-react/lib/(.*)$": "office-ui-fabric-react/lib-commonjs/$1"
```

Comment:
Just for my own knowledge, what does this do?

Reply:
The office-ui-fabric uses a different approach to export it's components and the Jest lib isn't able to load it. Take a look at this issue: [5733](https://github.com/microsoft/fluentui/issues/5733)